fancyshowcaseview/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.kt

```diff
@@ -35,22 +35,8 @@ import androidx.annotation.AttrRes
 import androidx.annotation.LayoutRes
 import androidx.annotation.RequiresApi
 import androidx.annotation.StyleRes
-import me.toptas.fancyshowcase.ext.AnimationEndListener
-import me.toptas.fancyshowcase.ext.attachedShowCase
-import me.toptas.fancyshowcase.ext.circularEnterAnimation
-import me.toptas.fancyshowcase.ext.circularExitAnimation
-import me.toptas.fancyshowcase.ext.globalLayoutListener
-import me.toptas.fancyshowcase.ext.rootView
-import me.toptas.fancyshowcase.internal.AndroidProperties
-import me.toptas.fancyshowcase.internal.AnimationPresenter
-import me.toptas.fancyshowcase.internal.DeviceParamsImpl
-import me.toptas.fancyshowcase.internal.FadeOutAnimation
-import me.toptas.fancyshowcase.internal.FancyImageView
-import me.toptas.fancyshowcase.internal.FocusedView
-import me.toptas.fancyshowcase.internal.Presenter
-import me.toptas.fancyshowcase.internal.Properties
-import me.toptas.fancyshowcase.internal.SharedPrefImpl
-import me.toptas.fancyshowcase.internal.getStatusBarHeight
+import me.toptas.fancyshowcase.ext.*
+import me.toptas.fancyshowcase.internal.*
```

Comment:
Not sure if its a style thing but at my work we usually don't allow blanket * for the whole class.  Not sure of this repos style guide